roles/cis/tasks/3/verify/3.5.1.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: "3.5.1 Verify if DCCP is disabled (Not Scored)"
+# Verifying the DCCP disablement settings in /etc/modprobe.d/modprobe.conf
```

Comment:
Please follow the following pattern for this type of verification
- name: 3.5.1 Verify if DCCP is disabled (Not Scored)
  lineinfile:
      path: "{{ modprobe_conf }}"
      regexp: "{{ dccp_settings_expected }}"
      state: absent
  check_mode: yes
  register: presence
- fail:
      msg: 3.5.1 failed to verify if DCCP is disabled, settings of DCCP is not expected in {{ modprobe_conf }}
  when: presence is changed

Reply:
Fixed!